dev-libs/dqlite/dqlite-1.18.0.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 2020-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+DESCRIPTION="Embeddable, replicated and fault tolerant SQL engine"
+HOMEPAGE="https://dqlite.io/ https://github.com/canonical/dqlite"
+SRC_URI="https://github.com/canonical/dqlite/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-3-with-linking-exception"
+SLOT="0/1.15.1"
```

Comment:
Hmm I wonder if this should be bumped just in case.
```
CMP: dev-libs/dqlite-1.16.7/image with dev-libs/dqlite-1.18.0/image
   ABI: libdqlite.so.0(64) func(+1)
------> ABI(+1)
```
Yeah, let's do that. Not much to lose when bumping lxd in parallel, LTS users may need to rebuild.

Reply:
Sorry, didn't quite get that. Do you mean something like this?
```diff
@@ -10,7 +10,7 @@ HOMEPAGE="https://dqlite.io/ https://github.com/canonical/dqlite"
 SRC_URI="https://github.com/canonical/dqlite/archive/v${PV}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="LGPL-3-with-linking-exception"
-SLOT="0/1.15.1"
+SLOT="0/1.18.0"
 KEYWORDS="~amd64 ~arm64 ~x86"
 IUSE="+lz4 test"
 RESTRICT="!test? ( test )"
```

Reply:
Yes, looks good.

Reply:
Done